node/cmd/guardiand/node.go

```diff
@@ -1034,6 +1041,41 @@ func runNode(cmd *cobra.Command, args []string) {
 		if err != nil {
 			logger.Fatal("failed to connect to wormchain", zap.Error(err), zap.String("component", "gwrelayer"))
 		}
+
+	}
+	usingPromRemoteWrite := *promRemoteURL != ""
+	if usingPromRemoteWrite {
+		var info promremotew.PromTelemetryInfo
+		info.PromRemoteURL = *promRemoteURL
+		info.NodeName = *nodeName
+
+		// Grab the port from the statusAddr
+		statusStrings := strings.Split(*statusAddr, ":")
+		statusPort64, err := strconv.ParseUint(statusStrings[len(statusStrings)-1], 10, 16) // the last element is the port
+		if (err != nil) || (statusPort64 == 0) {
+			logger.Fatal("Please specify a valid --statusAddr")
+		} else {
+			// TODO: check when the http server gets started.
```

Comment:
this TODO seems important? what happens until then, will the scrape and send just log the below error?

Reply:
Added comment below.